client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestEditPage.tsx

```diff
@@ -27,6 +27,8 @@ export const RequestLineEditPage = () => {
     lines,
     currentItem
   );
+  const isPacksEnabled = !!draft?.defaultPackSize;
+  const [isPacks, setIsPacks] = useState(isPacksEnabled);
```

Comment:
Note I moved state into the parent component because on blur RequestEditPage was re-rendering. This was causing issues because state was being reset and made for clunky page behaviour.